deploy/mixed-firmware/x86_64_x11ssh_qemu_linuxboot.defconfig

```diff
@@ -123,6 +123,7 @@ CONFIG_NET_SCHED=y
 CONFIG_NET_CLS_CGROUP=y
 CONFIG_CGROUP_NET_PRIO=y
 # CONFIG_WIRELESS is not set
+CONFIG_NET_9P=y
```

Comment:
Do you remember where in the kernel's menuconfig you have enabled this settings?

Reply:
Why do we need that. Also the paths change from kernel to kernel. You'd have to search for them anyway

Reply:
you are right with the paths, but I somhow wat to keep track of what kernel option is important for feature xy. Take a look at `deploy/mixed-firmware/README.md` "x86_64_linuxboot_config"

Reply:
And that didn't help either. I'd say, we document the significant options as `CONFIG_OPTION_NAME` and refer to the search function in `make menuconfig`

Reply:
Yes I will do and update that part.